ui/src/ensembler/components/modal/DeleteEnsemblerModal.js

```diff
@@ -54,40 +54,43 @@ export const DeleteEnsemblerModal = ({
   const modalClosed = () => {
     setCanDeleteEnsembler(true)
     setEnsemblerUsedByCurrentRouterVersion(false)
+    setEnsemblerUsedByActiveRouterVersion(false)
+    setEnsemblerUsedByActiveEnsemblingJob(false)
     setDeleteConfirmation("")
   }
 
   return (
     <ConfirmationModal
       title="Delete Ensembler"
       onCancel={() => modalClosed()}
-      onConfirm={submitForm}
+      onConfirm={(arg) => {submitForm(arg); modalClosed()}}
       isLoading={isLoading}
       content={
         <div>
           {canDeleteEnsembler ? (
             <div>
               <p>
-              You are about to delete Ensembler <b>{ensembler.name}</b>. This action cannot be undone. 
+              You are about to delete the Ensembler <b>{ensembler.name}</b>. This action <b>cannot</b> be undone.
               </p>
-              To confirm, please type "<b>{ensembler.name}</b>" in the box below
+              To confirm, please type "<b>{ensembler.name}</b>" in the box below:
               <EuiFieldText     
                 fullWidth            
                 placeholder={ensembler.name}
                 value={deleteConfirmation}
                 onChange={(e) => setDeleteConfirmation(e.target.value)}
                 isInvalid={deleteConfirmation !== ensembler.name} />              
             </div>
-          ) : ensemblerUsedByCurrentRouterVersion ? (
-            <div>
-              You cannot delete this ensembler because it is associated with a router version that is currently being used by a router
-              <br/> <br/> If you still wish to delete this ensembler, please <b>Deploy</b> another version on this router.
-            </div>
           ) : (
             <div>
-              You cannot delete this ensembler because there are <b>Active Router Versions</b> or <b>Ensembling Jobs</b> that use this ensembler. 
-              <br/> <br/> If you still wish to delete this ensembler, please <b>Undeploy</b> router versions and <b>Terminate</b> ensembling jobs that use this ensembler.
-            </div> 
+              You cannot delete this ensembler because it is:
+              <ul>
+                {ensemblerUsedByCurrentRouterVersion &&
+                  <li>associated with one or more router versions <b>currently</b> used by one or more routers
+                    {ensemblerUsedByActiveRouterVersion && ", and"}</li>}
+                {ensemblerUsedByActiveRouterVersion &&
+                  <li>used by one or more <b>Active Router Versions</b> or <b>Ensembling Jobs</b></li>}
+              </ul>
+            </div>
           )}
           {/* Only show The Ensembling Table if ensembler is not used by current router version */}
           {!ensemblerUsedByCurrentRouterVersion && (
```

Comment:
In case that the data in the modal becomes too large, should we make the content scrollable? Per [this issue](https://github.com/elastic/eui/discussions/4701), this should work automatically if wrapping the contents with `EuiModalBody`.

Reply:
Hmm I think it's already scrollable right? Unless you were referring to something else? 🤔 Like thee tables being scrollable?
![ezgif com-video-to-gif](https://github.com/caraml-dev/turing/assets/36802364/8e48b7eb-f6d5-4c37-8d01-0bff24fd5287)

Reply:
Oh I didn't realize this. This is great. 👍🏽